src/PKSim.Infrastructure/ORM/DAS/DAS.cs

```diff
@@ -19,6 +19,12 @@ public class DAS : IDisposable
 
       public DbConnection Connection { get; private set; }
 
+      public DAS()
+      {
+         // Initialize the provider factory to Microsoft.Data.Sqlite by default
+         _providerFactory = Microsoft.Data.Sqlite.SqliteFactory.Instance;
```

Comment:
why do we use the full namespace here? 
Also why do we instantiate in constgructor ?
This DAS the way it's written does not work with anything else but sqlite anymore. Should we change the code to simplify?